backend/src/Edna/Library/DB/Query.hs

```diff
@@ -148,19 +164,23 @@ insertCompound compoundName = do
 -- Test methodology
 --------------------------
 
-getMethodologyById :: SqlId 'MethodologyId -> Edna (Maybe TestMethodologyRec)
+-- | Get methodology by its name. Return nothing if there is no such methodology.
+getMethodologyById :: MethodologyId -> Edna (Maybe TestMethodologyRec)
 getMethodologyById (SqlId methodologyId) = runSelectReturningOne' $
   lookup_ (esTestMethodology ednaSchema) $ TestMethodologyId $ SqlSerial methodologyId
 
+-- | Get methodology by its name. Return nothing if there is no such methodology.
 getMethodologyByName :: Text -> Edna (Maybe TestMethodologyRec)
 getMethodologyByName name = runSelectReturningOne' $ select $ do
   methodologies <- all_ $ esTestMethodology ednaSchema
   guard_ (LDB.tmName methodologies ==. val_ name)
   pure methodologies
 
+-- | Get all methodologies
 getMethodologies :: Edna [TestMethodologyRec]
 getMethodologies = runSelectReturningList' $ select $ all_ $ esTestMethodology ednaSchema
 
+-- | Insert methodology and return its DB value
```

Comment:
I would add that it fails in case methodology with this name already exists. Same for `insertProject`.